AndroidSDKCore/src/main/java/com/leanplum/Leanplum.java

```diff
@@ -921,6 +938,16 @@ private static void applyContentInResponse(JSONObject response, boolean alwaysAp
         response.optJSONObject(Constants.Keys.REGIONS));
     List<Map<String, Object>> variants = JsonConverter.listFromJsonOrDefault(
         response.optJSONArray(Constants.Keys.VARIANTS));
+    VariantDebugInfo variantDebugInfo = null;
+    try {
+      String variantDebugInfoString = response.optString(Constants.Keys.VARIANT_DEBUG_INFO);
+      if (variantDebugInfoString != null) {
+        Gson gson = new Gson();
+        variantDebugInfo = gson.fromJson(variantDebugInfoString, VariantDebugInfo.class);
+      }
+    } catch (Throwable t) {
```

Comment:
Avoid catching throwable, instead catch specific exception that `fromJson` may throw.

Reply:
What's the reasoning behind this?
My goal here is to not block the rest of the leanplum response for a debug-info object. I could catch a list of particular exceptions, but my priority is to not block the execution here.